### PR TITLE
Update the jsdoc comments to modern syntax - Part 6

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
jsdoc part 6

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/control-bar/text-track-controls/captions-button.js [hunk 1]
```diff
@@ -8,24 +8,32 @@ import CaptionSettingsMenuItem from './caption-settings-menu-item.js';
...
+   * Creates an instance of this class.
+   *
+   * @param {Player} player
+   *        The `Player` that this class should be attached to.
+   *
+   * @param {Object} [options]
+   *        The key/value store of player options.
+   *
+   * @param {Component~ReadyCallback} [ready]
+   *        The functino to call when this component is ready.
```
Comment (misteroneill) [inline src/js/control-bar/text-track-controls/captions-button.js]: Typo - "functino"

File: src/js/control-bar/text-track-controls/subtitles-button.js [hunk 2]
```diff
@@ -7,32 +7,53 @@ import Component from '../../component.js';
...
+   * Creates an instance of this class.
+   *
+   * @param {Player} player
+   *        The `Player` that this class should be attached to.
+   *
+   * @param {Object} [options]
+   *        The key/value store of player options.
+   *
+   * @param {Component~ReadyCallback} [ready]
+   *        The functino to call when this component is ready.
```
Comment (misteroneill) [inline src/js/control-bar/text-track-controls/subtitles-button.js]: Typo - "functino"

File: src/js/control-bar/text-track-controls/descriptions-button.js [hunk 3]
```diff
@@ -8,14 +8,22 @@ import * as Fn from '../../utils/fn.js';
...
+   * Creates an instance of this class.
+   *
+   * @param {Player} player
+   *        The `Player` that this class should be attached to.
+   *
+   * @param {Object} [options]
+   *        The key/value store of player options.
+   *
+   * @param {Component~ReadyCallback} [ready]
+   *        The functino to call when this component is ready.
```
Comment (misteroneill) [inline src/js/control-bar/text-track-controls/descriptions-button.js]: Typo - "functino"

File: src/js/control-bar/time-controls/remaining-time-display.js [hunk 4]
```diff
@@ -47,11 +53,15 @@ class RemainingTimeDisplay extends Component {
   }
 
   /**
-   * Update remaining time display
+   * Update remaining time display.
+   *
+   * @param {EventTarget~Event} [event]
+   *        The `timeupdate` or `durationchange` event taht caused this to run.
```
Comment (misteroneill) [inline src/js/control-bar/time-controls/remaining-time-display.js]: Typo - "taht"

File: src/js/utils/events.js [hunk 5]
```diff
@@ -336,11 +353,18 @@ export function off(elem, type, fn) {
...
- * @param  {Object} [hash] data hash to pass along with the event
- * @return {Boolean=} Returned only if default was prevented
- * @method trigger
+ * @param {Element|Object} elem
+ *        Element to trigger an event on
+ *
+ * @param {EventTarget~Event|string} event
+ *        A string (the type) or an event object with a type attribute
+ *
+ * @param {Object} [hash] data
```
Comment (misteroneill) [inline src/js/utils/events.js]: Looks like "data" should be on the next line. And capitalized.

**Comment by BrandonOCasey** [comment]
@misteroneill Changes made

**Pull request merged**